daemon/emer-network-send-provider.c

```diff
@@ -191,15 +191,19 @@ reset_network_send_data (EmerNetworkSendProvider *self)
                   error->message);
       g_error_free (error);
     }
+
+  priv->send_number = 0;
+  priv->data_cached = TRUE;
 }
 
-static gboolean
+static void
 read_network_send_data (EmerNetworkSendProvider *self)
 {
   EmerNetworkSendProviderPrivate *priv =
     emer_network_send_provider_get_instance_private (self);
+
   if (priv->data_cached)
```

Comment:
I propose that we initialize members explicitly if and only if upstream refuses to mention that `g_object_new` zeroes out members of the private struct in its official documentation, which would suggest that they consider it an implementation detail that may be changed on a whim in future releases. Regardless, it's a pre-existing issue if it's an issue at all. @plward11 do you have any other comments on this pull request, or can we merge it?